### PR TITLE
merge firestore-api-changes to master

### PR DESCRIPTION
**Pull request by rsgowman**

### CODE REVIEW

File: Firestore/Example/SwiftBuildTest/main.swift [hunk 1]
```diff
@@ -217,6 +219,15 @@ func readDocument(at docRef: DocumentReference) {
   }
 }
 
+func readDocumentWithOptions(at docRef: DocumentReference) {
```
Comment (wilhuff) [inline Firestore/Example/SwiftBuildTest/main.swift]: Perhaps should now be named `readDocumentWithSource`?

File: Firestore/Example/Tests/Integration/API/FIRSourceTests.mm [hunk 2]
```diff
@@ -0,0 +1,648 @@
+/*
```
Comment (wilhuff) [inline Firestore/Example/Tests/Integration/API/FIRSourceTests.mm]: File should probably be renamed `FIRFirestoreSourceTests.mm` (sorry I missed this in your sed change).

File: Firestore/Example/SwiftBuildTest/main.swift (hunk 1 above)
Comment (rsgowman) [reply Firestore/Example/SwiftBuildTest/main.swift]: done

File: Firestore/Example/Tests/Integration/API/FIRSourceTests.mm (hunk 2 above)
Comment (rsgowman) [reply Firestore/Example/Tests/Integration/API/FIRSourceTests.mm]: done.

**Pull request merged**